<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnSupplierDevLogin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>supplierDev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>supplierPro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playDev">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>PlayDev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoginPlayPro">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>PlayPro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearEdit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>ClearEdit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSupplierWindow">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>supplier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSettingWin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>setting</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>7</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="textEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
